etc/ci/performance/credential.json.example

```diff
@@ -0,0 +1,6 @@
+{
+  "client_id":"your_id",
+  "secret": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
```

Comment:
I would prefer to pass these as environment variables, e.g. `TREEHERDER_CLIENT_ID` and `TREEHERDER_CLIENT_SECRET`.